index.html

```diff
@@ -64,6 +62,39 @@ <h2>
     <p>
       TBW
     </p>
+    <p id="credential-type-examples">
+      There are many types of digital credential that can be conveyed using
+      this API. Some of these types include:
+    </p>
+    <ul>
+      <li>
+        a driver's license issued by a testing facility
+      </li>
+      <li>
```

Comment:
I'm hesitant to include these at the moment because we haven't really looked at supporting these and I don't think any implementer supports these yet... though these could be supported, I'm worried about set expectations.

Reply:
I moved many of these examples down from the text that already existed in the abstract; I was attempting to preserve text that already existed. If you want to remove it, that's fine, though the utility of the API becomes less interesting for a variety of stakeholders that are watching this work closely.

Reply:
+1 to marcos. I understand you just moved the text, but mentioning passports in this file, either here or in the intro could be sensitive.

Reply:
Reduced the list down to the only credential type we have consensus on right now: 06271a72f5fcfbef716f61d49574b0de9449d5d0. Preserved the full list in https://github.com/WICG/digital-identities/issues/85#issuecomment-1963077832.